open_spiel/games/chess.cc

```diff
@@ -139,14 +138,14 @@ Color PlayerToColor(Player p) {
   return static_cast<Color>(p);
 }
 
-Action MoveToAction(const Move& move) {
+Action MoveToAction(const Move& move, const int board_size) {
```

Comment:
No need to declare int as const:

For a function parameter passed by value, const has no effect on the caller, thus is not recommended in function declarations. See https://abseil.io/tips/109